tensorflow/lite/kernels/activations.cc

```diff
@@ -328,6 +328,19 @@ TfLiteStatus PreluPrepare(TfLiteContext* context, TfLiteNode* node) {
   return kTfLiteOk;
 }
 
+template <typename T>
+void QuantizedRelu(const TfLiteTensor* input, TfLiteTensor* output) {
+  ActivationParams params;
+  params.activation_type = FusedActivationFunctionType::kRelu;
+  params.quantized_activation_min =
```

Comment:
Could you please explain why is zero point used in min but not in max?

Reply:
@jianlijianli , thanks for the review, i have updated the code, kindly check.

Regards
Amit